### DISCUSSION TITLE
Help: Boolean search tips in build_query

### DISCUSSION THREAD

**Discussion by cssetupman** [cjbarrie/academictwitteR#368] [General]
### Describe the solution you'd like

I want to count # of tweets that include ("artificial intelligence" or "algorithm*") AND ("disappointing"). 
I added the asterisk after algorithm to collect any tweets that include algorithm such as algorithmic. 

aiconv <- build_query(
  query = c("artificial intelligence","algorithm*","disappointing"),
  is_retweet = FALSE,
  lang = "en")

aiconv

AIcount<-count_all_tweets(query = aiconv,
                 start_tweets = "2022-11-29T00:00:00Z",
                 end_tweets = "2022-12-01T00:00:00Z",
                 is_retweet = FALSE,
                 lang = "en")

Then, I got the error code 400 as below.
Error in make_query(url = endpoint_url, params = params, bearer_token = bearer_token,  : 
  something went wrong. Status code: 400

Can anyone instruct me how to use this boolean search rule in build_query command.
I can only find single search term examples here: https://github.com/cjbarrie/academictwitteR/blob/master/vignettes/academictwitteR-build.Rmd
Thank you so much for your help.

**Comment by cjbarrie** [comment]
This is not a feature request so has been converted to a Discussion. Please read the documents on what your status codes mean.
